internal/backend/remote-state/s3/backend.go

```diff
@@ -750,9 +756,9 @@ func (b *Backend) Configure(obj cty.Value) tfdiags.Diagnostics {
 		// Note: we don't need to read env variables explicitly because they are read implicitly by aws-sdk-base-go:
```

Comment:
Should we remove or alter this comment? Because we are reading the env variables explicitly